src/discord_utils.py

```diff
@@ -31,6 +31,26 @@ async def try_set_pin(message: Message, pin_state: bool) -> None:
         print("Altering message pin state failed:", e)
 
 
+def attachment_local_filepath(message_id: int, attachment: Attachment) -> str:
+    """Build local filepath with provided Message ID & Attachment object for saving Attachments locally"""
+    filepath = path.join(
+        config.attachment_directory_filepath,
+        "{}.{}{}".format(
+            message_id,
+            attachment.id,
+            os.path.splitext(attachment.filename)[1],
+        ),
+    )
+    return filepath
+
+
+def is_valid_media(attachment_content_type: str) -> bool:
```

Comment:
Could you include a docstring to explain what this function does (and what is considered "valid media")?

Something along the lines of:

```suggestion
def is_valid_media(attachment_content_type: str) -> bool:
"""
Returns whether an attachment's content type is considered "media".

Valid media types are those that start with "audio" or "video".

Args:
    attachment_content_type: A content type (aka MIME type).
Returns:
    True if the content type is considered to be media, otherwise False.
```